File: CTDK_Python/047/_047.py
from math import sqrt


n = int(input("Nhap n: "))

s = 0
i = 1
while(i <= n):
    s = s + sqrt(1 + 1 / (i * i) + 1 / (i + 1)**2)
    i = i + 1

print("Tong la: ", s)

File: LTHAM/047/047.cpp
#include <iostream>
#include <cmath>
using namespace std;

float Tong(int);

int main()
{
	int n;
	cout << "Nhap n: ";
	cin >> n;
	
	cout << "Tong la: " << Tong(n) << endl;
	return 0;
}

float Tong(int nn)
{
	float s = 0;
	int i = 1;
	while (i <= nn)
	{
		s = s + sqrt(1 + 1/(pow(i,2)) + 1/(pow(i+1,2)));
		i++;
	}
	return s;
}